sparrow-py/.darglint

```diff
@@ -0,0 +1,3 @@
+[darglint]
+strictness = short
+docstring_style=numpy
```

Comment:
I think the python client code is predominantly Google style. We should decide on one style and go with that. 

Also this is another python tool. We should also make a selection of what python tools we are to use and be consistent across python code in the repo